src/Core/Domain/Product/ValueObject/Isbn.php

```diff
@@ -38,7 +38,9 @@ class Isbn
     /**
      * Valid ISBN regex pattern
      */
-    public const VALID_PATTERN = '/^[0-9-]{0,32}$/';
+    public const VALID_PATTERN = '/^(?:ISBN(?:-1[03])?:? )?(?=[0-9X]{10}
```

Comment:
If you're using https://www.oreilly.com/library/view/regular-expressions-cookbook/9781449327453/ch04s13.html you can maybe specify it 
or better, use the defined pattern with comments:
```
^
(?:ISBN(?:-1[03])?:?\ )?  # Optional ISBN/ISBN-10/ISBN-13 identifier.
(?=                       # Basic format pre-checks (lookahead):
  [0-9X]{10}$             #   Require 10 digits/Xs (no separators).
 |                        #  Or:
  (?=(?:[0-9]+[-\ ]){3})  #   Require 3 separators
  [-\ 0-9X]{13}$          #     out of 13 characters total.
 |                        #  Or:
  97[89][0-9]{10}$        #   978/979 plus 10 digits (13 total).
 |                        #  Or:
  (?=(?:[0-9]+[-\ ]){4})  #   Require 4 separators
  [-\ 0-9]{17}$           #     out of 17 characters total.
)                         # End format pre-checks.
(?:97[89][-\ ]?)?         # Optional ISBN-13 prefix.
[0-9]{1,5}[-\ ]?          # 1-5 digit group identifier.
[0-9]+[-\ ]?[0-9]+[-\ ]?  # Publisher and title identifiers.
[0-9X]                    # Check digit.
$
```

Reply:
I can't add comments or use a lot of line breaks the test fails; I added all that in the comment

Reply:
You can, by using concatenation